README.md

```diff
@@ -0,0 +1,55 @@
+# Webots Animation Action
+
+This GitHub action creates a Webots animation of a simulation and publishes it to GitHub pages.
+
+<p align="center">
+  <img src="./assets/cover.png">
+</p>
+
+
+After each commit, Webots simulation will be recorded and published to `<username>.github.io/<repository>` as an X3D animation.
+In your browser, you can move around and zoom while the animation is playing.
+
+## Workflow
+
+Here is a simple GitHub workflow snippet which utilizes the action:
+```yaml
+name: Record animation
+
+on: [push]
```

Comment:
```suggestion
on:
  push:
    branches: 
      - master
```
This is just a suggestion, but the user may want that the animation(s) represent the state of master.

Reply:
Good idea!